'kubectl kudo plan history' fails on non-existent instances

**What happened**:
CLI produced a stacktrace to the output when running `kubectl kudo plan history --instance nonexistent`:
```
$> kubectl kudo plan history --instance nonexistent
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x198 pc=0x1e353ec]

goroutine 1 [running]:
github.com/kudobuilder/kudo/pkg/kudoctl/cmd/plan.planHistory(0x3454240, 0x3454ea0, 0x8, 0x7ffeefbff99c)
	/Users/alenavarkockova/go/src/github.com/kudobuilder/kudo/pkg/kudoctl/cmd/plan/plan_history.go:56 +0x14c
github.com/kudobuilder/kudo/pkg/kudoctl/cmd/plan.RunHistory(0xc000431180, 0x3454240, 0x3454ea0, 0x0, 0x0)
	/Users/alenavarkockova/go/src/github.com/kudobuilder/kudo/pkg/kudoctl/cmd/plan/plan_history.go:32 +0xc9
github.com/kudobuilder/kudo/pkg/kudoctl/cmd.NewPlanHistoryCmd.func1(0xc000431180, 0xc0004c5b20, 0x0, 0x2, 0x0, 0x0)
	/Users/alenavarkockova/go/src/github.com/kudobuilder/kudo/pkg/kudoctl/cmd/plan.go:39 +0x40
github.com/spf13/cobra.(*Command).execute(0xc000431180, 0xc0004c5ac0, 0x2, 0x2, 0xc000431180, 0xc0004c5ac0)
	/Users/alenavarkockova/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826 +0x460
github.com/spf13/cobra.(*Command).ExecuteC(0xc000298f00, 0x0, 0x2240fa0, 0xc00009c058)
	/Users/alenavarkockova/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/alenavarkockova/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
main.main()
	/Users/alenavarkockova/go/src/github.com/kudobuilder/kudo/cmd/kubectl-kudo/main.go:24 +0x27
```
**What you expected to happen**:
Same behavior as for `plan status`:
```
$>kubectl kudo plan status --instance nonexistent
Error: client Error: instances.kudo.dev "nonexistent" not found
```
**How to reproduce it (as minimally and precisely as possible)**:
By running `kubectl kudo plan history --instance nonexistent`
**Environment**:
- Kubernetes version:
  ```
  Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.1", GitCommit:"d647ddbd755faf07169599a625faf302ffc34458", GitTreeState:"clean", 
  BuildDate:"2019-10-02T23:49:20Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"darwin/amd64"}
  Server Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.3", GitCommit:"2d3c76f9091b6bec110a5e63777c332469e0cba2", GitTreeState:"clean", 
  BuildDate:"2019-08-19T11:05:50Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
  ```
- Kudo version:
   ```
  KUDO Version: version.Info{GitVersion:"0.7.3", GitCommit:"d81e57e7", BuildDate:"2019-10-08T11:32:32Z", GoVersion:"go1.13", Compiler:"gc", Platform:"darwin/amd64"}
  ```

--- a/pkg/controller/instance/instance_controller_test.go
+++ b/pkg/controller/instance/instance_controller_test.go
@@ -84,7 +84,7 @@ func instancePlanFinished(key client.ObjectKey, planName string, c client.Client
 	i := &v1alpha1.Instance{}
 	err := c.Get(context.TODO(), key, i)
 	if err != nil {
-		fmt.Printf("%w", err)
+		fmt.Printf("%v", err)
 		return false
 	}
 	return i.Status.PlanStatus[planName].Status.IsFinished()

--- a/pkg/kudoctl/cmd/plan/plan_history.go
+++ b/pkg/kudoctl/cmd/plan/plan_history.go
@@ -38,7 +38,7 @@ func planHistory(options *Options, settings *env.Settings) error {
 
 	kc, err := kudo.NewClient(settings.Namespace, settings.KubeConfig)
 	if err != nil {
-		fmt.Printf("Unable to create kudo client to talk to kubernetes API server %w", err)
+		fmt.Printf("Unable to create kudo client to talk to kubernetes API server %v", err)
 		return err
 	}
 	instance, err := kc.GetInstance(options.Instance, namespace)

